content/use-cases/use-an-escrow-as-a-smart-contract.md

```diff
@@ -0,0 +1,137 @@
+
+
+
+<span class="use-case-step-num">{{n.next()}}</span>
+## Define the terms of the smart contract
+
+To create the escrow as a smart contract, the participants must first define the terms of the contract. In this scenario, the participants need to agree on the following details.
+
+- **Should the escrow disallow fulfillment until a specific time?**
+
+      While this is an option, the participants agree that it is unnecessary for their escrow. For conditionally-held escrows, enabling this option doesn't provide any additional security, since whether the escrow can be finished still depends entirely on whether the party planner (oracle) publishes the fulfillment before the expiration.
+
+- **Should the escrow expire?**
+
+      Absolutely yes. The participants agree that the escrow should expire after 12 noon the day after the party. This gives the party band (receiver) enough time to finish the escrow, after the party planner verifies that they fulfilled their end of the contract and publishes the cryptographic fulfillment. After expiration, the locked XRP returns to the party host's (sender's) account.
+
+      If the participants don't allow the escrow to expire and the party planner doesn't release the condition, the XRP stays locked in the escrow forever.
+
+- **How much XRP should the escrow lock up and potentially pay?**
+
+      The participants agree that the escrow should lock up and potentially pay 2000 XRP, which is the party band's fee.
+
+- **From which XRP Ledger account should the escrow lock up XRP for potential payment to the party band?**
+
+      The participants agree that the escrow should lock up and potentially pay XRP out of the party host's XRP Ledger account.
+
+- **Which XRP Ledger account should the escrow potentially pay XRP to?**
+
+      The participants agree that the escrow should potentially pay XRP to the party band's XRP Ledger account.
+
+
+
+<span class="use-case-step-num">{{n.next()}}</span>
+## [Generate a condition and a fulfillment](send-a-conditionally-held-escrow.html#1-generate-condition-and-fulfillment) (oracle)
+
+Because participants want to create a conditionally-held escrow to provide the smart contract, they need a condition value and a fulfillment value. In this scenario, the participant that creates these values is the neutral party planner (oracle).
+
+The party planner generates the condition and fulfillment values. The party planner provides the condition value to the party host, who creates the escrow. The part planner also provides the condition to the party band so that they know that this is the right condition.
+
+The party planner must keep the fulfillment value a secret. Anyone can use the condition and fulfillment values to finish the escrow, but in this scenario, it will most likely be the party band who will finish the escrow because they're the ones who are motivated to get paid.
```

Comment:
> Anyone can use the condition and fulfillment values to finish the escrow, but in this scenario, it will most likely be the party band who will finish the escrow because they're the ones who are motivated to get paid.

I suggest breaking this up into two sentences and rewording to avoid future tense:

> Anyone can use the condition and fulfillment values to finish the escrow. Most often, the receiver finishes the escrow because they're the ones who are motivated to get paid.